From 7c0dcead14d79244a7bba06053369da17289503d Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Fri, 1 Nov 2019 18:22:13 +0100
Subject: [PATCH 08/26] Ugly but secure: Set /dev/uinput as 0666 to fix
 fingerprint sensor on some devices

cf https://github.com/phhusson/device_phh_treble/pull/122/commits/e000d69c286b6686777ea6f1867f379e30273e48
This is safe because even though it's 0666, its SELinux policy is very
tight, and only bluetooth HAL, shell (and fingerprint HAL on Xiaomi) can
access it.

Change-Id: Id374e781957927d5604cb96c7a39b3fb28b3a6c5
---
 rootdir/ueventd.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rootdir/ueventd.rc b/rootdir/ueventd.rc
index 0b7ffb8ea6..e06e453b11 100644
--- a/rootdir/ueventd.rc
+++ b/rootdir/ueventd.rc
@@ -54,7 +54,7 @@ subsystem dma_heap
 
 # these should not be world writable
 /dev/uhid                 0660   uhid       uhid
-/dev/uinput               0660   uhid       uhid
+/dev/uinput               0666   uhid       uhid
 /dev/rtc0                 0640   system     system
 /dev/tty0                 0660   root       system
 /dev/graphics/*           0660   root       graphics
-- 
2.34.1

